### PR TITLE
release notes 2023.01.15

### PR DESCRIPTION
**Pull request by phealy**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+* Behavior Changes
+  * AKS clusters deployed in [dual-stack (IPv4/IPv6) mode](https://learn.microsoft.com/azure/aks/configure-kubenet-dual-stack) and utilizing [instance-level public IP addresses](https://learn.microsoft.com/azure/aks/use-node-public-ips) will now receive both IPv4 and IPv6 public IP addresses.
+  * The `nosharesock` option has been added to the default Azure Files dynamic storage class to address [this GitHub issue](https://github.com/kubernetes-sigs/azurefile-csi-driver/issues/1137).
+* Component Updates
+  * Azure Monitor Managed Prometheus has been updated to [release 01-11-2023](https://github.com/Azure/prometheus-collector/blob/07ba2a930b61ea6f1deae91fb16b05da19cf1f1a/RELEASENOTES.md#release-01-11-2023).
+  * Azure disk csi driver v2 has been updated to [v2.0.0-beta.10](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases/tag/v2.0.0-beta.10).
+  * AKS Windows 2022 image has been updated to [20348.1487.230111](vhd-notes/AKSWindows/2022/20348.1487.230111.txt).
+  * AKS Windows 2019 image has been updated to [17763.3887.230111]](vhd-notes/AKSWindows/2019/17763.3887.230111.txt).
```
Comment (phealy) [inline CHANGELOG.md:27]:
```suggestion
  * AKS Windows 2019 image has been updated to [17763.3887.230111](vhd-notes/AKSWindows/2019/17763.3887.230111.txt).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+
+* Starting in API version 2023-01-01, the following API changes will occur:
+  * The name for the AKS cluster SKU Name `Basic` will change to `Base`. `Basic` will be removed in the 2023-07-01 API version.
+  * The name for the AKS cluster SKU Tier `Paid` will change to `Standard`. `Paid` will be removed in the 2023-07-01 API version.
+* AKS began pod security policy deprecation on 2022-11-01 API. The [pod security policy](https://learn.microsoft.com/azure/aks/use-pod-security-policies) will be removed completely on 2023-06-01 API with AKS 1.25 version or higher. You can migrate pod security policy to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) before the deprecation deadline.
+
+### Release notes
+
+* Features
+  * Starting with Kubernetes 1.26:
```
Comment (palma21) [inline CHANGELOG.md:17]: Having in mind 1.26 is not out yet. This is an announcement in the previous section to be moved here when 1.26 is out in preview/GA?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+    * Some AKS labels will be deprecated. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Behavior Changes
+  * AKS clusters deployed in [dual-stack (IPv4/IPv6) mode](https://learn.microsoft.com/azure/aks/configure-kubenet-dual-stack) and utilizing [instance-level public IP addresses](https://learn.microsoft.com/azure/aks/use-node-public-ips) will now receive both IPv4 and IPv6 public IP addresses.
+  * The `nosharesock` option has been added to the default Azure Files dynamic storage class to address [this GitHub issue](https://github.com/kubernetes-sigs/azurefile-csi-driver/issues/1137).
+* Component Updates
+  * Azure Monitor Managed Prometheus has been updated to [release 01-11-2023](https://github.com/Azure/prometheus-collector/blob/07ba2a930b61ea6f1deae91fb16b05da19cf1f1a/RELEASENOTES.md#release-01-11-2023).
+  * Azure disk csi driver v2 has been updated to [v2.0.0-beta.10](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases/tag/v2.0.0-beta.10).
```
Comment (palma21) [inline CHANGELOG.md:25]: This is not released?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,34 @@
...
+  * AKS Windows 2022 image has been updated to [20348.1487.230111](vhd-notes/AKSWindows/2022/20348.1487.230111.txt).
+  * AKS Windows 2019 image has been updated to [17763.3887.230111](vhd-notes/AKSWindows/2019/17763.3887.230111.txt).
+  * Azure Policy will be updated to [GateKeeper 3.11](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.11.0) on Feb 20th for AKS 1.24+.
+  * Containerd on Windows has been updated to [v1.6.14](https://github.com/containerd/containerd/releases/tag/v1.6.14).
+  * Open Service Mesh Addon has been updated to [v1.2.3](https://github.com/openservicemesh/osm/releases/tag/v1.2.3) for clusters running AKS 1.24+.
+
```
Comment (palma21) [inline CHANGELOG.md:31]: Linux images?

**Review by palma21 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,34 @@
...
 
+## Release 2023-01-15
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Starting in API version 2023-01-01, the following API changes will occur:
+  * The name for the AKS cluster SKU Name `Basic` will change to `Base`. `Basic` will be removed in the 2023-07-01 API version.
+  * The name for the AKS cluster SKU Tier `Paid` will change to `Standard`. `Paid` will be removed in the 2023-07-01 API version.
```
Comment (shashankbarsin) [inline CHANGELOG.md:11]: per team discussion, let's time this announcement with the blog having details of each tier to provide clarity

File: CHANGELOG.md (hunk 3 above)
Comment (phealy) [reply CHANGELOG.md:25]: Which one of these, sorry? The CSI v2? @allyford?

File: CHANGELOG.md (hunk 4 above)
Comment (phealy) [reply CHANGELOG.md:31]: Not sure where those went, they were in there originally (and I added the files...). Thanks.

File: CHANGELOG.md (hunk 2 above)
Comment (phealy) [reply CHANGELOG.md:17]: moved

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,34 @@
...
+
+### Release notes
+
+* Features
+  * Starting with Kubernetes 1.26:
+    * HostProcess Containers will be GA
+    * Some AKS labels will be deprecated. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Behavior Changes
+  * AKS clusters deployed in [dual-stack (IPv4/IPv6) mode](https://learn.microsoft.com/azure/aks/configure-kubenet-dual-stack) and utilizing [instance-level public IP addresses](https://learn.microsoft.com/azure/aks/use-node-public-ips) will now receive both IPv4 and IPv6 public IP addresses.
+  * The `nosharesock` option has been added to the default Azure Files dynamic storage class to address [this GitHub issue](https://github.com/kubernetes-sigs/azurefile-csi-driver/issues/1137).
```
[the remark this answers is not in the document]
Comment (phealy) [reply CHANGELOG.md:22]: added behavior change text

File: CHANGELOG.md (hunk 3 above)
Comment (palma21) [reply CHANGELOG.md:21]: yes, @olsenme can you clarify?

Comment (palma21) [reply CHANGELOG.md:21]: @phealy we have an announcement for it previously?

**Review by palma21 — APPROVED** [review]

**Review by shashankbarsin — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (phealy) [reply CHANGELOG.md:21]: Meagan is OOO until 1/30, pulling for now.

Comment (phealy) [reply CHANGELOG.md:21]:
```suggestion
```

**Pull request merged**